shells/pipes-shell/web/web.js

```diff
@@ -24,6 +24,8 @@ import {RamSlotComposer} from '../../lib/components/ram-slot-composer.js';
 import {findContainers} from '../source/lib/utils.js';
 import {initPipe, initArcs} from '../source/pipe.js';
 import {smokeTest} from '../source/smoke.js';
+// fetch override that supports file
+import '../source/lib/fetch-local.js';
 
 console.log(`${version} -- ${storage}`);
 
```

Comment:
is this `composerFactory` still used? i think the one declared in pipe.js is now used instead.

Reply:
I tried to make this PR by cobbling together bits from other branches, and it looks like I crossed the streams. 

Sorry, I'll circle back when I fix it.